tasks/main.yml

```diff
@@ -180,13 +221,25 @@
   when:
     - ansible_distribution == "Alpine"
 
+- name: install bootstrap packages for OpenBSD
```

Comment:
Was it possible to use the single `install bootstrap packages` instead of this custom task for OpenBSD only?